src/doodle/doodleController.js

```diff
@@ -71,19 +98,22 @@ export class DoodleController {
   }
 
   render() {
-    const setLines = newLines => {
-      this.lines = newLines;
+    const setLines = lines => {
+      this.newLines = lines;
     };
     render(
-      <DoodleCanvas
-        attachedElement={this._container}
-        size={this._size}
-        tool={this._tool}
-        active={this._doodleable}
-        lines={this._lines}
-        setLines={setLines}
-      />,
-      document.body
+      <Fragment>
```

Comment:
I love Fragments, one of the best things to be added to react. FYI, you can use <></> instead of \<Fragment>\</Fragment> (unless their preact is out of date, then I'm not sure)